### DISCUSSION TITLE
Loosing HTML signature.

### DISCUSSION THREAD

**Discussion by PeopleInside** [the-djmaze/snappymail#747] [General]
Hi, since sometimes I see that, on account settings and identity I loose all accounts HTML signature saved.
I don't know if this happen every time I update or what is causing this loosing.

Any idea why this happen? I suppose when I save HTML signature on identity nothing should made me loosing it.

**Comment by PeopleInside** [comment]
I think I understood what is the problem and steps to reproduce.

I logged into my webmail for update my HTML signature on all account... ( I need do one by one).
Logged in Account one then settings, identity and no HTML signature was showed.
Pasted my signature

When the browser is closed and you return back signature will work for new email but if you still go on settings, identity no HTML signature will be showed. Seems after closing the browser signature are lost in settings identity

**Reply by MI-KY** [reply]
Hi,
the signatures are saved in a file named "identities" if I'm not completly wrong... So the browser itself should not have to do something with the signature. But somehow I have already seen a similar thing: when I edit a signature i saved my changes. When I entered again into the configuration of the identity (directly entred again without closing browser etc.) sometimes the signature was gone...

Could you maybe try if this happens also to you? Maybe this is "just" a problem in the visualization of the signature inside the editor...

**Comment by PeopleInside** [comment]
Yes the issue seems something like this. In the editor signature are no more visible.

**Comment by MI-KY** [comment]
Hi,

so I have made a look and this seems to affect also my installation. When I insert something in the signature field and hit the save button this is written to the file `identities`.
But when I reload the page and enter again into the settings the field is empty (but the signature still is activly used in the mails and also existing in the `identities` file.
Seems that SnappyMail somehow mixes up something when getting the identities of the user... 🙂 .
Could you please create an issue for this problem?

**Reply by PeopleInside** [reply]
Done, thank you for your help!
https://github.com/the-djmaze/snappymail/issues/750

**Reply by MI-KY** [reply]
You're welcome 🙂
